org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/format/experimental/XMLFormatterExperimentalTest.java

```diff
@@ -678,37 +679,55 @@ public void testCommentMultiLineContent() throws BadLocationException {
 
 	@Test
 	public void testCommentLongWrap() throws BadLocationException {
-		String content = "<a>\n" + //
-				"  Content <!-- comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment -->"
+		String content = "<a> Content <!-- comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment comment -->"

```

Comment:
Instead of adjusting `content`, consider maybe set joinContent to true? (or making a duplicate of this test with joinContent=true)